libdnf/utils/GzFile.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef LIBDNF_GZFILE_HPP
+#define LIBDNF_GZFILE_HPP
+
+#include <zlib.h>
```

Comment:
please, no. libsolv is something what already handles such things. Stop duplicating functionality.